api/controllers/Irc_EntryController.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
 
   index: (req, res) => res.ok,
 
-  process_heartbeat: (request_body) =>
+  heartbeatProcess: (request_body) =>
```

Comment:
The danger with this naming scheme is that the `sails-controller-driven-routing` would allow you to use an http verb to execute a method you didn't want to expose.
so `curl -X PROCESS http://.../irc_entry/heartbeat` would in principle allow you to execute what is actually a private method